programs/ssl/ssl_test_lib.h

```diff
@@ -246,18 +246,21 @@ int key_opaque_alg_parse( const char *arg, const char **alg1, const char **alg2
  *
  *
  * \param alg1          input string opaque key algorithm #1
- * \param alg1          input string opaque key algorithm #2
+ * \param alg2          input string opaque key algorithm #2
  * \param psa_alg1      output PSA algorithm #1
  * \param psa_alg2      output PSA algorithm #2
  * \param usage         output key usage
+ * \param pkey          key used to set default psa algorithm/usage
```

Comment:
Perhaps we could pass only the key type here instead of the whole key? Or at least make it `const` so that it's clear we're not modifying it?

Reply:
Yes, just force-pushed and changed key to key type.